Dockerfile-ppc64le

```diff
@@ -0,0 +1,44 @@
+ARG quay_expiration=never
```

Comment:
Small nit: I'd prefer the filename to be Dockerfile.ppc64le instead of with the '-' but not a blocker.